media/inf/keyboard.inf

```diff
@@ -208,3 +208,8 @@ StdMfg = "(Standartinė klaviatūra)"
 HID.KeyboardDevice    = "HID 键盘"
 StdMfg = "(标准键盘)"
 *PNP0303.DeviceDesc = "IBM PC/AT 键盘"
+
+[Strings.0816]
+KeyboardClassName = "Teclados"
+StdMfg = "(Teclado padrão)"
+*PNP0303.DeviceDesc = "Teclado IBM PC/AT"
```

Comment:
`LocationOverride` and `HID.KeyboardDevice` are not translated, but it's ok to do later.